lte/gateway/python/magma/mobilityd/ip_allocator_factory.py

```diff
@@ -0,0 +1,66 @@
+import redis
+import logging
+from collections import defaultdict
+
+from magma.mobilityd.ip_descriptor import IPDesc
+from lte.protos.mconfig.mconfigs_pb2 import MobilityD
+
+from magma.mobilityd import mobility_store as store
+
+from .ip_allocator_dhcp import IPAllocatorDHCP
+from .ip_allocator_static import IpAllocatorStatic
+from .ip_descriptor_map import IpDescriptorMap
+from .uplink_gw import UplinkGatewayInfo
+
+class IPAllocatorFactory:
+    def __init__(self, ip_allocator_type, mobilityd_conf, redis_config):
+        """ Creates IP allocator, states and sid maps in redis/memory flavors
+        """
+        persist_to_redis = mobilityd_conf.get('persist_to_redis', True)
```

Comment:
not related to the changes, but let's update the default to False to follow what the default we have on the yml file and also default stateful behavior